packages/esm-patient-notes-app/src/notes/visit-notes.resource.ts

```diff
@@ -89,10 +87,11 @@ function formatDiagnoses(diagnoses: Array<DiagnosisData>): Array<Diagnosis> {
 
 function mapDiagnosisProperties(diagnosis: DiagnosisData): Diagnosis {
   return {
-    concept: diagnosis.concept,
-    conceptReferenceTermCode: getConceptReferenceTermCode(diagnosis.concept.conceptMappings).conceptReferenceTerm.code,
+    uuid: diagnosis.uuid,
+    concept: diagnosis.name,
+    conceptReferenceTermCode: getConceptReferenceTermCode(diagnosis.mappings).conceptReferenceTerm.display,
     primary: false,
-    confirmed: false,
+    confirmed: true,
```

Comment:
Why do we default to assuming a diagnosis is confirmed? ("Confirmed" in this context generally means "we have a lab test that proves the patient has x")